dbt/dbt_project.yml

```diff
@@ -22,6 +22,18 @@ clean-targets:
   - "target"
   - "dbt_packages"
 
+vars:
+  # Start year for testing data using test_qc_* tagged dbt tests. Typically
+  # this should be the current year, since errors in past data cannot usually
+  # be amended once records are closed. Set as an integer for compatibility
+  # with comparison operators and SQL's BETWEEN
+  test_qc_year_start: 2023
```

Comment:
I figure it's no big deal to change these once per year. This also gives us the flexibility to test prior years if we ever want to.